tests/testthat/_snaps/listing_egl01.md

```diff
@@ -5,62 +5,62 @@
-                                                                       Week 20    141          55.00              -7.00             363.00             -32.00             889.00             286.00    
-                                                                       Week 24    171          74.00              12.00             444.00              49.00             679.00              76.00    
-                                                                       Week 26    183          72.00              10.00             366.00             -29.00            531.00/L            -72.00    
-                                                                       Week 4     29           76.00              14.00             448.00              53.00            536.00/L            -67.00    
-                                                                       Week 6     43           67.00               5.00             441.00              46.00            560.00/L            -43.00    
-                                                                       Week 8     57           69.00               7.00            483.00/H             88.00            493.00/L           -110.00    
-                      701/1234                  69/M/WHITE             Week 12    100          65.00             -11.00             391.00            -145.00            540.00/L             50.00    
-      —————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+                                                                                          Heart Rate Result                    QT Duration Result                    RR Duration Result                 
+                                                                                  Study     (BEATS/MIN);        Heart Rate          (msec);          QT Duration          (msec);          RR Duration  
+      Treatment   Center/Subject ID            Age/Sex/Race             Visit      Day     Range:(40-100)     Change from BL    Range:(350-450)     Change from BL    Range:(600-1500)    Change from BL
+      ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+       Placebo        701/1015                  63/F/WHITE             Baseline    NA            73.67                NA            456.67/H                NA            519.00/L                NA    
```

Comment:
@shajoezhu it seems like the rows in ADEG where `ANL01FL == "Y"` have missing data (all `NA`s) in most other columns, which leads to the all missing "Study Day" column seen in this listing.

Reply:
i think this `study day` should be added back in the admial data 

@manciniedoardo , I was wondering if the average would be computed across multiple days? @jeffreyad @bundfussr

@khatril fyi

Reply:
@shajoezhu , the average is computed by date (`ADT`). Unfortunately, we don't keep `ADY` for the derived records. I'll reopen the issue to add it.

Reply:
@shajoezhu @bundfussr I just added `ADY` to summary records in template.  Thanks https://github.com/pharmaverse/admiral/pull/2572

Reply:
thanks so much @bundfussr @jeffreyad

Reply:
This is looking great now after the updates in pharmaverseadam. Thanks all!